### PR TITLE
ci config for nodejs wrapper

### PR DESCRIPTION
**Pull request by farskipper**
Waiting on #616

### CODE REVIEW

File: Jenkinsfile.ci [hunk 1]
```diff
@@ -234,6 +234,7 @@ def linuxTesting(file, env_name, network_name) {
 
         sh "cp libindy/target/debug/libindy.so wrappers/java/lib"
```
Comment (Artemkaaas) [inline Jenkinsfile.ci]: You also should update docker files ubuntu and amazon placed here libindy/ci. 
After that, on your local machine, you can check your steps inside docker.
Not it doesn't work for me

File: Jenkinsfile.ci [hunk 2]
```diff
@@ -338,6 +341,21 @@ def linuxPythonTesting(env_name, network_name, testEnv) {
...
 }
 
+def linuxNodejsTesting(env_name, network_name, testEnv) {
+    unstash name: "LibindyNodejsSO${env_name}"
+    dir('wrappers/nodejs') {
+        testEnv.inside("--network=${network_name}") {
+            echo "${env_name} Libindy Test: Test nodejs wrapper"
+
+            sh '''
+                npm run cp-include
```
Comment (Artemkaaas) [inline Jenkinsfile.ci]: I added in ubuntu.dockerfile the following lines:
```
RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
RUN apt-get install -y nodejs
RUN npm install -g node-gyp
```
I am getting ```missing script: cp-include```  on executing this command ```npm run cp-include```

**Comment by farskipper** [comment]
@Artemkaaas Thanks. I made a fix for Jenkinsfile.ci and added nodejs to the ci dockerfiles

**Comment by jovfer** [comment]
#830 little updated ci pipline and fixed tests to satisfy the latest Libindy

**Pull request closed without merging**